pkg/machine/qemu/machine.go

```diff
@@ -630,18 +644,21 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	}
 
 	v.waitAPIAndPrintInfo(forwardState, forwardSock)
-
+	//	v.Starting = false
 	return nil
 }
 
 func (v *MachineVM) checkStatus(monitor *qmp.SocketMonitor) (machine.Status, error) {
 	// this is the format returned from the monitor
 	// {"return": {"status": "running", "singlestep": false, "running": true}}
 
+	// REMEMBER
```

Comment:
The Alamo?

Reply:
Agh thought I got all of my comments, will do another sweep